addons/play-json/src/main/scala/com/gilt/handlebars/scala/play_json/package.scala

```diff
@@ -0,0 +1,90 @@
+package com.gilt.handlebars.scala
+
+import play.api.libs.json._
+import com.gilt.handlebars.scala.helper.Helper
+
+package object play_json {
```

Comment:
Probably want to avoid underscores in package names. What do you think about alternatives?

`com.gilt.handlebars.scala.playjson` or
`com.gilt.handlebars.scala.play.json` 

??